webpack.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = (env) => {
 
   return {
     target: 'web',
-    entry: './src/client.ts',
+    entry: './src/index.ts',
```

Comment:
The reason the barrel file wasn't getting exported was... because we weren't exporting the barrel smh. Took me far too long to figure that out.